cdshooks-wih/pom.xml

```diff
@@ -31,14 +31,25 @@
   </properties>
 
  <dependencies>
-   	<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<scope>test</scope>
-		</dependency>
     <dependency>
-    	<groupId>io.elimu.a2d2</groupId>
-    	<artifactId>kie-based-services</artifactId>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+    	<groupId>org.kie</groupId>
+    	<artifactId>kie-api</artifactId>
```

Comment:
Should this also be test?

Reply:
@boxwala it depends on kie-api to understand the WorkItemHandler interface. All the WIH components we have depend on it, directly or indirectly

Reply:
Thank you for clarifying @marianbuenosayres .